tests/test_cli.py

```diff
@@ -678,6 +678,37 @@ def test_run_non_universal_newline(self):
         self.assertEqual(
             (ctx.returncode, ctx.stdout, ctx.stderr), (1, expected_out, ''))
 
+    def test_run_list_files(self):
+        self.maxDiff = None
```

Comment:
This line shouldn't be commited.

Reply:
At some point I saw a message suggesting to add `maxDiff` but I am not seeing it anymore and diffs as in https://github.com/adrienverge/yamllint/actions/runs/3854993441/jobs/6571327795 works fine :+1: 

Removing! Thanks :bow:

Reply:
Oh I just saw that very warning again:

```
$ python3 -m unittest tests/test_cli.py
..........F......................
======================================================================
FAIL: test_run_list_files (tests.test_cli.CommandLineTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/peter/devel/yamllint/tests/test_cli.py", line 685, in test_run_list_files
    self.assertEqual(ctx.stdout, (
AssertionError: '/tmp[20 chars]cglf/c.yaml\n/tmp/yamllint-tests-vpxhcglf/empt[327 chars]ml\n' != '/tmp[20 chars]cglf/a.yaml\n/tmp/yamllint-tests-vpxhcglf/c.ya[327 chars]ml\n'
Diff is 773 characters long. Set self.maxDiff to None to see it.

----------------------------------------------------------------------
Ran 33 tests in 0.201s

FAILED (failures=1)
```

Reply:
It looks like this line came back during your last push :wink:

Reply:
Shooot! Fixed :heavy_check_mark: 

Sorry about that - I had to use it to debug some failures I was seeing :see_no_evil: